docs/docs/en/guide/task/dms.md

```diff
@@ -0,0 +1,107 @@
+# DMS Node
+
+## Overview
+
+[Pytorch](https://pytorch.org) is a mainstream Python machine learning library.
+
+[AWS Database Migration Service (AWS DMS)](https://aws.amazon.com/cn/dms) helps you migrate databases to AWS quickly and securely. 
+The source database remains fully operational during the migration, minimizing downtime to applications that rely on the database. 
+The AWS Database Migration Service can migrate your data to and from the most widely used commercial and open-source databases.
+
+DMS task plugin can help users to create and start DMS tasks in DolphinScheduler more conveniently.
+
+Contains two features:
+- Create DMS task and start DMS task
+- Restart DMS task
+
+We can create DMS task and start DMS task in two ways:
+- Use interface
+- Use JSON data
+
+DolphinScheduler will track the status of the DMS task and set the status to successfully completed when the DMS task is completed. Except for the CDC task without end time.
+
+So, if the `migrationType` is `cdc` or `full-load-and-cdc`, `cdcStopPosition` not be set, DolphinScheduler will set the status to successfully after the DMS task starts successfully.
+
+## Create Task
+
+- Click `Project Management -> Project Name -> Workflow Definition`, and click the `Create Workflow` button to enter the DAG editing page.
+- Drag <img src="../../../../img/tasks/icons/dms.png" width="15"/> from the toolbar to the canvas.
+
+## Task Example
+
+The task plugin picture is as follows
+
+**Create and start DMS task by interface**
+
+![dms](../../../../img/tasks/demo/dms_create_and_start.png)
+
+
+**Restart DMS task by interface**
+
+![dms](../../../../img/tasks/demo/dms_restart.png)
+
+
+**Create and start DMS task by JSON data**
+
+![dms](../../../../img/tasks/demo/dms_create_and_start_json.png)
+
+**Restart DMS task by JSON data**
+
+![dms](../../../../img/tasks/demo/dms_restart_json.png)
+
+
+
+### First, introduce some general parameters of DolphinScheduler
```

Comment:
We have put those default parameters in a separate doc for better maintainability. See: https://github.com/apache/dolphinscheduler/pull/11776

Reply:
done